File: csharp/PirateCannons.cs
using System;
using System.Collections.Generic;


namespace DailyTraining
{
  public class PirateCannon
  {
    public static string CannonsReady(Dictionary<string, string> gunners)
    {
      if (gunners.ContainsValue("nay"))
      {
        return "Shiver me timbers!";
      }
      return "Fire!";
    }
    
  }
}

File: python/PirateCannons.py
def cannons_ready(gunners):
  for pirate in gunners.values():
    if pirate == "nay":
      return "Shiver me timbers!"
  return "Fire!"

print(cannons_ready({'Mike':'aye','Joe':'aye','Johnson':'aye','Peter':'aye'}))
# should return "Fire"

print(cannons_ready({'Mike':'aye','Joe':'nay','Johnson':'aye','Peter':'aye'}))
# should return "Shiver me timbers"